using-valtimo/document/custom-case-columns.md

```diff
@@ -0,0 +1,162 @@
+# Custom case columns
+
+Each case comes with a default case overview with four standard columns. 'Reference number', 'Created by', 'Created on',
+and 'Last modified'. These columns can be adjusted. Any key defined in a document can be used as a column in a case
+overview.
+
+| <!-- -->                                                      |
+|---------------------------------------------------------------|
+| ![Default case detail list](img/default-case-detail-list.png) |
+| <center>Default case overview</center>                        |
```

Comment:
interesting way of adding a caption. i need to see how this is displayed by gitbook before i can make up my mind whether i like it.